Taskfile.yaml

```diff
@@ -311,6 +311,12 @@ tasks:
     cmds:
       - poetry run pytest --junitxml=pytest.xml -k playground tests/notebooks
 
+  test-reset:
```

Comment:
a small utility which i used when trying to run integration tests

Reply:
Curious under which situation will we need to run this?

Reply:
i removed it for now, but there were some situations where my db state seemed corrupted, so i was just running this to reset, and restart my local test env in one command